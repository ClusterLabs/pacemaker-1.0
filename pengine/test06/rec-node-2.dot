 digraph "g" {
"all_stopped" -> "rsc1_start_0 node2" [ style = bold]
"all_stopped" -> "rsc2_start_0 node2" [ style = bold]
"all_stopped" -> "rsc3_start_0 node2" [ style = bold]
"all_stopped" -> "rsc4_start_0 node2" [ style = bold]
"all_stopped" -> "rsc5_start_0 node2" [ style = bold]
"all_stopped" -> "rsc6_start_0 node2" [ style = bold]
"all_stopped" [ style=bold color="green" fontcolor="orange"  ]
"group1_running_0" [ style=bold color="green" fontcolor="orange" ]
"group1_start_0" -> "group1_running_0" [ style = bold]
"group1_start_0" -> "rsc3_start_0 node2" [ style = bold]
"group1_start_0" -> "rsc4_start_0 node2" [ style = bold]
"group1_start_0" [ style=bold color="green" fontcolor="orange" ]
"group1_stop_0" -> "group1_start_0" [ style = bold]
"group1_stop_0" -> "group1_stopped_0" [ style = bold]
"group1_stop_0" -> "rsc3_stop_0" [ style = bold]
"group1_stop_0" -> "rsc4_stop_0" [ style = bold]
"group1_stop_0" [ style=bold color="green" fontcolor="orange"  ]
"group1_stopped_0" -> "group1_start_0" [ style = bold]
"group1_stopped_0" [ style=bold color="green" fontcolor="orange"  ]
"group2_running_0" [ style=bold color="green" fontcolor="orange" ]
"group2_start_0" -> "group2_running_0" [ style = bold]
"group2_start_0" -> "rsc5_start_0 node2" [ style = bold]
"group2_start_0" -> "rsc6_start_0 node2" [ style = bold]
"group2_start_0" [ style=bold color="green" fontcolor="orange" ]
"group2_stop_0" -> "group2_start_0" [ style = bold]
"group2_stop_0" -> "group2_stopped_0" [ style = bold]
"group2_stop_0" -> "rsc5_stop_0" [ style = bold]
"group2_stop_0" -> "rsc6_stop_0" [ style = bold]
"group2_stop_0" [ style=bold color="green" fontcolor="orange"  ]
"group2_stopped_0" -> "group2_start_0" [ style = bold]
"group2_stopped_0" [ style=bold color="green" fontcolor="orange"  ]
"probe_complete node2" -> "probe_complete" [ style = bold]
"probe_complete node2" [ style=bold color="green" fontcolor="black" ]
"probe_complete" -> "group1_start_0" [ style = bold]
"probe_complete" -> "group1_stop_0" [ style = bold]
"probe_complete" -> "group2_start_0" [ style = bold]
"probe_complete" -> "group2_stop_0" [ style = bold]
"probe_complete" -> "rsc1_start_0 node2" [ style = bold]
"probe_complete" -> "rsc2_start_0 node2" [ style = bold]
"probe_complete" -> "rsc3_start_0 node2" [ style = bold]
"probe_complete" -> "rsc3_stop_0" [ style = bold]
"probe_complete" -> "rsc4_start_0 node2" [ style = bold]
"probe_complete" -> "rsc4_stop_0" [ style = bold]
"probe_complete" -> "rsc5_start_0 node2" [ style = bold]
"probe_complete" -> "rsc5_stop_0" [ style = bold]
"probe_complete" -> "rsc6_start_0 node2" [ style = bold]
"probe_complete" -> "rsc6_stop_0" [ style = bold]
"probe_complete" [ style=bold color="green" fontcolor="orange" ]
"rsc1_monitor_0 node2" -> "probe_complete node2" [ style = bold]
"rsc1_monitor_0 node2" [ style=bold color="green" fontcolor="black" ]
"rsc1_start_0 node2" [ style=bold color="green" fontcolor="black" ]
"rsc2_monitor_0 node2" -> "probe_complete node2" [ style = bold]
"rsc2_monitor_0 node2" [ style=bold color="green" fontcolor="black" ]
"rsc2_start_0 node2" [ style=bold color="green" fontcolor="black" ]
"rsc3_monitor_0 node2" -> "probe_complete node2" [ style = bold]
"rsc3_monitor_0 node2" [ style=bold color="green" fontcolor="black" ]
"rsc3_start_0 node2" -> "group1_running_0" [ style = bold]
"rsc3_start_0 node2" -> "rsc4_start_0 node2" [ style = bold]
"rsc3_start_0 node2" [ style=bold color="green" fontcolor="black" ]
"rsc3_stop_0" -> "all_stopped" [ style = bold]
"rsc3_stop_0" -> "group1_stopped_0" [ style = bold]
"rsc3_stop_0" [ style=bold color="green" fontcolor="orange"  ]
"rsc4_monitor_0 node2" -> "probe_complete node2" [ style = bold]
"rsc4_monitor_0 node2" [ style=bold color="green" fontcolor="black" ]
"rsc4_start_0 node2" -> "group1_running_0" [ style = bold]
"rsc4_start_0 node2" [ style=bold color="green" fontcolor="black" ]
"rsc4_stop_0" -> "all_stopped" [ style = bold]
"rsc4_stop_0" -> "group1_stopped_0" [ style = bold]
"rsc4_stop_0" -> "rsc3_stop_0" [ style = bold]
"rsc4_stop_0" [ style=bold color="green" fontcolor="orange"  ]
"rsc5_monitor_0 node2" -> "probe_complete node2" [ style = bold]
"rsc5_monitor_0 node2" [ style=bold color="green" fontcolor="black" ]
"rsc5_start_0 node2" -> "group2_running_0" [ style = bold]
"rsc5_start_0 node2" -> "rsc6_start_0 node2" [ style = bold]
"rsc5_start_0 node2" [ style=bold color="green" fontcolor="black" ]
"rsc5_stop_0" -> "all_stopped" [ style = bold]
"rsc5_stop_0" -> "group2_stopped_0" [ style = bold]
"rsc5_stop_0" [ style=bold color="green" fontcolor="orange"  ]
"rsc6_monitor_0 node2" -> "probe_complete node2" [ style = bold]
"rsc6_monitor_0 node2" [ style=bold color="green" fontcolor="black" ]
"rsc6_start_0 node2" -> "group2_running_0" [ style = bold]
"rsc6_start_0 node2" [ style=bold color="green" fontcolor="black" ]
"rsc6_stop_0" -> "all_stopped" [ style = bold]
"rsc6_stop_0" -> "group2_stopped_0" [ style = bold]
"rsc6_stop_0" -> "rsc5_stop_0" [ style = bold]
"rsc6_stop_0" [ style=bold color="green" fontcolor="orange"  ]
"stonith node1" -> "all_stopped" [ style = bold]
"stonith node1" [ style=bold color="green" fontcolor="black" ]
"stonith_up" -> "stonith node1" [ style = bold]
"stonith_up" [ style=bold color="green" fontcolor="orange"  ]
}
