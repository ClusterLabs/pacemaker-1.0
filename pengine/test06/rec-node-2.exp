 <transition_graph cluster-delay="60s" stonith-timeout="60s" failed-stop-offset="INFINITY" failed-start-offset="INFINITY" batch-limit="30" transition_id="0">
   <synapse id="0">
     <action_set>
       <rsc_op id="4" operation="monitor" operation_key="rsc1_monitor_0" on_node="node2" on_node_uuid="uuid2">
         <primitive id="rsc1" long-id="rsc1" class="heartbeat" type="apache"/>
         <attributes CRM_meta_op_target_rc="7" CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </rsc_op>
     </action_set>
     <inputs/>
   </synapse>
   <synapse id="1">
     <action_set>
       <rsc_op id="10" operation="start" operation_key="rsc1_start_0" on_node="node2" on_node_uuid="uuid2">
         <primitive id="rsc1" long-id="rsc1" class="heartbeat" type="apache"/>
         <attributes CRM_meta_name="start" CRM_meta_requires="quorum" CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </rsc_op>
     </action_set>
     <inputs>
       <trigger>
         <pseudo_event id="1" operation="all_stopped" operation_key="all_stopped"/>
       </trigger>
       <trigger>
         <pseudo_event id="2" operation="probe_complete" operation_key="probe_complete"/>
       </trigger>
     </inputs>
   </synapse>
   <synapse id="2">
     <action_set>
       <rsc_op id="5" operation="monitor" operation_key="rsc2_monitor_0" on_node="node2" on_node_uuid="uuid2">
         <primitive id="rsc2" long-id="rsc2" class="heartbeat" type="apache"/>
         <attributes CRM_meta_op_target_rc="7" CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </rsc_op>
     </action_set>
     <inputs/>
   </synapse>
   <synapse id="3">
     <action_set>
       <rsc_op id="11" operation="start" operation_key="rsc2_start_0" on_node="node2" on_node_uuid="uuid2">
         <primitive id="rsc2" long-id="rsc2" class="heartbeat" type="apache"/>
         <attributes CRM_meta_name="start" CRM_meta_requires="fencing" CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </rsc_op>
     </action_set>
     <inputs>
       <trigger>
         <pseudo_event id="1" operation="all_stopped" operation_key="all_stopped"/>
       </trigger>
       <trigger>
         <pseudo_event id="2" operation="probe_complete" operation_key="probe_complete"/>
       </trigger>
     </inputs>
   </synapse>
   <synapse id="4">
     <action_set>
       <pseudo_event id="14" operation="start" operation_key="group1_start_0">
         <attributes CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </pseudo_event>
     </action_set>
     <inputs>
       <trigger>
         <pseudo_event id="2" operation="probe_complete" operation_key="probe_complete"/>
       </trigger>
       <trigger>
         <pseudo_event id="16" operation="stop" operation_key="group1_stop_0"/>
       </trigger>
       <trigger>
         <pseudo_event id="17" operation="stopped" operation_key="group1_stopped_0"/>
       </trigger>
     </inputs>
   </synapse>
   <synapse id="5">
     <action_set>
       <pseudo_event id="15" operation="running" operation_key="group1_running_0">
         <attributes CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </pseudo_event>
     </action_set>
     <inputs>
       <trigger>
         <rsc_op id="12" operation="start" operation_key="rsc3_start_0" on_node="node2" on_node_uuid="uuid2"/>
       </trigger>
       <trigger>
         <rsc_op id="13" operation="start" operation_key="rsc4_start_0" on_node="node2" on_node_uuid="uuid2"/>
       </trigger>
       <trigger>
         <pseudo_event id="14" operation="start" operation_key="group1_start_0"/>
       </trigger>
     </inputs>
   </synapse>
   <synapse id="6">
     <action_set>
       <pseudo_event id="16" operation="stop" operation_key="group1_stop_0">
         <attributes CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </pseudo_event>
     </action_set>
     <inputs>
       <trigger>
         <pseudo_event id="2" operation="probe_complete" operation_key="probe_complete"/>
       </trigger>
     </inputs>
   </synapse>
   <synapse id="7">
     <action_set>
       <pseudo_event id="17" operation="stopped" operation_key="group1_stopped_0">
         <attributes CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </pseudo_event>
     </action_set>
     <inputs>
       <trigger>
         <pseudo_event id="16" operation="stop" operation_key="group1_stop_0"/>
       </trigger>
       <trigger>
         <pseudo_event id="28" operation="stop" operation_key="rsc3_stop_0"/>
       </trigger>
       <trigger>
         <pseudo_event id="29" operation="stop" operation_key="rsc4_stop_0"/>
       </trigger>
     </inputs>
   </synapse>
   <synapse id="8">
     <action_set>
       <rsc_op id="6" operation="monitor" operation_key="rsc3_monitor_0" on_node="node2" on_node_uuid="uuid2">
         <primitive id="rsc3" long-id="group1:rsc3" class="heartbeat" type="apache"/>
         <attributes CRM_meta_op_target_rc="7" CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </rsc_op>
     </action_set>
     <inputs/>
   </synapse>
   <synapse id="9">
     <action_set>
       <rsc_op id="12" operation="start" operation_key="rsc3_start_0" on_node="node2" on_node_uuid="uuid2">
         <primitive id="rsc3" long-id="group1:rsc3" class="heartbeat" type="apache"/>
         <attributes CRM_meta_name="start" CRM_meta_requires="quorum" CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </rsc_op>
     </action_set>
     <inputs>
       <trigger>
         <pseudo_event id="1" operation="all_stopped" operation_key="all_stopped"/>
       </trigger>
       <trigger>
         <pseudo_event id="2" operation="probe_complete" operation_key="probe_complete"/>
       </trigger>
       <trigger>
         <pseudo_event id="14" operation="start" operation_key="group1_start_0"/>
       </trigger>
     </inputs>
   </synapse>
   <synapse id="10">
     <action_set>
       <pseudo_event id="28" operation="stop" operation_key="rsc3_stop_0">
         <attributes CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </pseudo_event>
     </action_set>
     <inputs>
       <trigger>
         <pseudo_event id="2" operation="probe_complete" operation_key="probe_complete"/>
       </trigger>
       <trigger>
         <pseudo_event id="16" operation="stop" operation_key="group1_stop_0"/>
       </trigger>
       <trigger>
         <pseudo_event id="29" operation="stop" operation_key="rsc4_stop_0"/>
       </trigger>
     </inputs>
   </synapse>
   <synapse id="11">
     <action_set>
       <rsc_op id="7" operation="monitor" operation_key="rsc4_monitor_0" on_node="node2" on_node_uuid="uuid2">
         <primitive id="rsc4" long-id="group1:rsc4" class="heartbeat" type="apache"/>
         <attributes CRM_meta_op_target_rc="7" CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </rsc_op>
     </action_set>
     <inputs/>
   </synapse>
   <synapse id="12">
     <action_set>
       <rsc_op id="13" operation="start" operation_key="rsc4_start_0" on_node="node2" on_node_uuid="uuid2">
         <primitive id="rsc4" long-id="group1:rsc4" class="heartbeat" type="apache"/>
         <attributes CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </rsc_op>
     </action_set>
     <inputs>
       <trigger>
         <pseudo_event id="1" operation="all_stopped" operation_key="all_stopped"/>
       </trigger>
       <trigger>
         <pseudo_event id="2" operation="probe_complete" operation_key="probe_complete"/>
       </trigger>
       <trigger>
         <rsc_op id="12" operation="start" operation_key="rsc3_start_0" on_node="node2" on_node_uuid="uuid2"/>
       </trigger>
       <trigger>
         <pseudo_event id="14" operation="start" operation_key="group1_start_0"/>
       </trigger>
     </inputs>
   </synapse>
   <synapse id="13">
     <action_set>
       <pseudo_event id="29" operation="stop" operation_key="rsc4_stop_0">
         <attributes CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </pseudo_event>
     </action_set>
     <inputs>
       <trigger>
         <pseudo_event id="2" operation="probe_complete" operation_key="probe_complete"/>
       </trigger>
       <trigger>
         <pseudo_event id="16" operation="stop" operation_key="group1_stop_0"/>
       </trigger>
     </inputs>
   </synapse>
   <synapse id="14">
     <action_set>
       <pseudo_event id="20" operation="start" operation_key="group2_start_0">
         <attributes CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </pseudo_event>
     </action_set>
     <inputs>
       <trigger>
         <pseudo_event id="2" operation="probe_complete" operation_key="probe_complete"/>
       </trigger>
       <trigger>
         <pseudo_event id="22" operation="stop" operation_key="group2_stop_0"/>
       </trigger>
       <trigger>
         <pseudo_event id="23" operation="stopped" operation_key="group2_stopped_0"/>
       </trigger>
     </inputs>
   </synapse>
   <synapse id="15">
     <action_set>
       <pseudo_event id="21" operation="running" operation_key="group2_running_0">
         <attributes CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </pseudo_event>
     </action_set>
     <inputs>
       <trigger>
         <rsc_op id="18" operation="start" operation_key="rsc5_start_0" on_node="node2" on_node_uuid="uuid2"/>
       </trigger>
       <trigger>
         <rsc_op id="19" operation="start" operation_key="rsc6_start_0" on_node="node2" on_node_uuid="uuid2"/>
       </trigger>
       <trigger>
         <pseudo_event id="20" operation="start" operation_key="group2_start_0"/>
       </trigger>
     </inputs>
   </synapse>
   <synapse id="16">
     <action_set>
       <pseudo_event id="22" operation="stop" operation_key="group2_stop_0">
         <attributes CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </pseudo_event>
     </action_set>
     <inputs>
       <trigger>
         <pseudo_event id="2" operation="probe_complete" operation_key="probe_complete"/>
       </trigger>
     </inputs>
   </synapse>
   <synapse id="17">
     <action_set>
       <pseudo_event id="23" operation="stopped" operation_key="group2_stopped_0">
         <attributes CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </pseudo_event>
     </action_set>
     <inputs>
       <trigger>
         <pseudo_event id="22" operation="stop" operation_key="group2_stop_0"/>
       </trigger>
       <trigger>
         <pseudo_event id="30" operation="stop" operation_key="rsc5_stop_0"/>
       </trigger>
       <trigger>
         <pseudo_event id="31" operation="stop" operation_key="rsc6_stop_0"/>
       </trigger>
     </inputs>
   </synapse>
   <synapse id="18">
     <action_set>
       <rsc_op id="8" operation="monitor" operation_key="rsc5_monitor_0" on_node="node2" on_node_uuid="uuid2">
         <primitive id="rsc5" long-id="group2:rsc5" class="heartbeat" type="apache"/>
         <attributes CRM_meta_op_target_rc="7" CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </rsc_op>
     </action_set>
     <inputs/>
   </synapse>
   <synapse id="19">
     <action_set>
       <rsc_op id="18" operation="start" operation_key="rsc5_start_0" on_node="node2" on_node_uuid="uuid2">
         <primitive id="rsc5" long-id="group2:rsc5" class="heartbeat" type="apache"/>
         <attributes CRM_meta_name="start" CRM_meta_requires="fencing" CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </rsc_op>
     </action_set>
     <inputs>
       <trigger>
         <pseudo_event id="1" operation="all_stopped" operation_key="all_stopped"/>
       </trigger>
       <trigger>
         <pseudo_event id="2" operation="probe_complete" operation_key="probe_complete"/>
       </trigger>
       <trigger>
         <pseudo_event id="20" operation="start" operation_key="group2_start_0"/>
       </trigger>
     </inputs>
   </synapse>
   <synapse id="20">
     <action_set>
       <pseudo_event id="30" operation="stop" operation_key="rsc5_stop_0">
         <attributes CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </pseudo_event>
     </action_set>
     <inputs>
       <trigger>
         <pseudo_event id="2" operation="probe_complete" operation_key="probe_complete"/>
       </trigger>
       <trigger>
         <pseudo_event id="22" operation="stop" operation_key="group2_stop_0"/>
       </trigger>
       <trigger>
         <pseudo_event id="31" operation="stop" operation_key="rsc6_stop_0"/>
       </trigger>
     </inputs>
   </synapse>
   <synapse id="21">
     <action_set>
       <rsc_op id="9" operation="monitor" operation_key="rsc6_monitor_0" on_node="node2" on_node_uuid="uuid2">
         <primitive id="rsc6" long-id="group2:rsc6" class="heartbeat" type="apache"/>
         <attributes CRM_meta_op_target_rc="7" CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </rsc_op>
     </action_set>
     <inputs/>
   </synapse>
   <synapse id="22">
     <action_set>
       <rsc_op id="19" operation="start" operation_key="rsc6_start_0" on_node="node2" on_node_uuid="uuid2">
         <primitive id="rsc6" long-id="group2:rsc6" class="heartbeat" type="apache"/>
         <attributes CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </rsc_op>
     </action_set>
     <inputs>
       <trigger>
         <pseudo_event id="1" operation="all_stopped" operation_key="all_stopped"/>
       </trigger>
       <trigger>
         <pseudo_event id="2" operation="probe_complete" operation_key="probe_complete"/>
       </trigger>
       <trigger>
         <rsc_op id="18" operation="start" operation_key="rsc5_start_0" on_node="node2" on_node_uuid="uuid2"/>
       </trigger>
       <trigger>
         <pseudo_event id="20" operation="start" operation_key="group2_start_0"/>
       </trigger>
     </inputs>
   </synapse>
   <synapse id="23">
     <action_set>
       <pseudo_event id="31" operation="stop" operation_key="rsc6_stop_0">
         <attributes CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </pseudo_event>
     </action_set>
     <inputs>
       <trigger>
         <pseudo_event id="2" operation="probe_complete" operation_key="probe_complete"/>
       </trigger>
       <trigger>
         <pseudo_event id="22" operation="stop" operation_key="group2_stop_0"/>
       </trigger>
     </inputs>
   </synapse>
   <synapse id="24">
     <action_set>
       <pseudo_event id="1" operation="all_stopped" operation_key="all_stopped">
         <attributes crm_feature_set="3.0.1"/>
       </pseudo_event>
     </action_set>
     <inputs>
       <trigger>
         <crm_event id="25" operation="stonith" operation_key="stonith" on_node="node1" on_node_uuid="uuid1"/>
       </trigger>
       <trigger>
         <pseudo_event id="28" operation="stop" operation_key="rsc3_stop_0"/>
       </trigger>
       <trigger>
         <pseudo_event id="29" operation="stop" operation_key="rsc4_stop_0"/>
       </trigger>
       <trigger>
         <pseudo_event id="30" operation="stop" operation_key="rsc5_stop_0"/>
       </trigger>
       <trigger>
         <pseudo_event id="31" operation="stop" operation_key="rsc6_stop_0"/>
       </trigger>
     </inputs>
   </synapse>
   <synapse id="25">
     <action_set>
       <pseudo_event id="2" operation="probe_complete" operation_key="probe_complete">
         <attributes crm_feature_set="3.0.1"/>
       </pseudo_event>
     </action_set>
     <inputs>
       <trigger>
         <rsc_op id="3" operation="probe_complete" operation_key="probe_complete" on_node="node2" on_node_uuid="uuid2"/>
       </trigger>
     </inputs>
   </synapse>
   <synapse id="26" priority="1000000">
     <action_set>
       <rsc_op id="3" operation="probe_complete" operation_key="probe_complete" on_node="node2" on_node_uuid="uuid2">
         <attributes CRM_meta_op_no_wait="true" crm_feature_set="3.0.1"/>
       </rsc_op>
     </action_set>
     <inputs>
       <trigger>
         <rsc_op id="4" operation="monitor" operation_key="rsc1_monitor_0" on_node="node2" on_node_uuid="uuid2"/>
       </trigger>
       <trigger>
         <rsc_op id="5" operation="monitor" operation_key="rsc2_monitor_0" on_node="node2" on_node_uuid="uuid2"/>
       </trigger>
       <trigger>
         <rsc_op id="6" operation="monitor" operation_key="rsc3_monitor_0" on_node="node2" on_node_uuid="uuid2"/>
       </trigger>
       <trigger>
         <rsc_op id="7" operation="monitor" operation_key="rsc4_monitor_0" on_node="node2" on_node_uuid="uuid2"/>
       </trigger>
       <trigger>
         <rsc_op id="8" operation="monitor" operation_key="rsc5_monitor_0" on_node="node2" on_node_uuid="uuid2"/>
       </trigger>
       <trigger>
         <rsc_op id="9" operation="monitor" operation_key="rsc6_monitor_0" on_node="node2" on_node_uuid="uuid2"/>
       </trigger>
     </inputs>
   </synapse>
   <synapse id="27">
     <action_set>
       <pseudo_event id="24" operation="stonith_up" operation_key="stonith_up">
         <attributes crm_feature_set="3.0.1"/>
       </pseudo_event>
     </action_set>
     <inputs/>
   </synapse>
   <synapse id="28">
     <action_set>
       <crm_event id="25" operation="stonith" operation_key="stonith" on_node="node1" on_node_uuid="uuid1">
         <attributes CRM_meta_on_node="node1" CRM_meta_on_node_uuid="uuid1" CRM_meta_stonith_action="reboot" crm_feature_set="3.0.1"/>
       </crm_event>
     </action_set>
     <inputs>
       <trigger>
         <pseudo_event id="24" operation="stonith_up" operation_key="stonith_up"/>
       </trigger>
     </inputs>
   </synapse>
 </transition_graph>

