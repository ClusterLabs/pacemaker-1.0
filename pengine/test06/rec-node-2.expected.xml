<?xml version="1.0" encoding="UTF-8"?>
<cib admin_epoch="0" epoch="1" num_updates="1" dc-uuid="0" have-quorum="true" remote-tls-port="0" validate-with="pacemaker-1.0">
  <configuration>
    <crm_config><cluster_property_set id="cib-bootstrap-options"><nvpair id="nvpair" name="stonith-enabled" value="true"/><nvpair id="nvpair" name="no-quorum-policy" value="stop"/></cluster_property_set></crm_config>
    <nodes>
      <node id="uuid1" uname="node1" type="member"/>
      <node id="uuid2" uname="node2" type="member"/>
    </nodes>
    <resources>
      <primitive id="rsc1" class="heartbeat" type="apache"><meta_attributes id="primitive-rsc1.meta"/>
	<operations>
	  <op name="stop" interval="0" id="stop-1" on-fail="block"/>
	  <op name="start" interval="0" id="start-1" requires="quorum"/>
	</operations>
      </primitive>
      <primitive id="rsc2" class="heartbeat" type="apache"><meta_attributes id="primitive-rsc2.meta"/>
	<operations>
	  <op name="stop" interval="0" id="stop-2" on-fail="block"/>
	  <op name="start" interval="0" id="start-2" requires="fencing"/>
	</operations>
      </primitive>
      <group id="group1"><meta_attributes id="group-group1.meta"/>
	<primitive id="rsc3" class="heartbeat" type="apache"><meta_attributes id="primitive-rsc3.meta"/>
	  <operations>
	    <op name="start" interval="0" id="op.auto-5" requires="quorum"/>
	  </operations>
	</primitive>
	<primitive id="rsc4" class="heartbeat" type="apache"><meta_attributes id="primitive-rsc4.meta"/></primitive>
      </group>
      <group id="group2"><meta_attributes id="group-group2.meta"/>
	<primitive id="rsc5" class="heartbeat" type="apache"><meta_attributes id="primitive-rsc5.meta"/>
	  <operations>
	    <op name="start" interval="0" id="op.auto-6" requires="fencing"/>
	  </operations>
	</primitive>
	<primitive id="rsc6" class="heartbeat" type="apache"><meta_attributes id="primitive-rsc6.meta"/></primitive>
      </group>
    </resources>
    <constraints/>
  </configuration>
  <status>
    <node_state id="uuid2" ha="active" uname="node2" crmd="online" join="member" expected="member" in_ccm="true"/>
  </status>
</cib>
